autoria.py

```diff
@@ -0,0 +1,253 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+from typing import Union
+
+
+class RiaAPI:
+    """Python auto.ria.com API.
+
+    Python implementation of API intended for calulating
+    average used car prices that are sold on http://auto.ria.com
+    """
+
+    def __init__(self):
+        """Initialize a class."""
+        self._base_url = 'http://api.auto.ria.com'
+
+    def _make_request(self, url: str) -> Union[dict, list]:
+        """Send get request and return data in JSON.
+
+        Args:
+            url - url returning needed data
+
+        Returns:
+            JSON-formatted response text.
+        """
+        response = requests.get(self._base_url + url)
+        if response.status_code == 200:
+            return json.loads(response.text)
+        else:
+            print('Ups! Something went wrong in processing of {}'.format(url))
```

Comment:
Thank you very much for the explanation!
I was almost sure that the print is not the best way here, but didn't figure out better way :)